build/webpack/webpack.components.entries.json

```diff
@@ -1 +1,14 @@
-{}
\ No newline at end of file
+{
```

Comment:
Use finsemble.webpack.json and put it in in the component folder. This makes things more modular.